app/assets/javascripts/darkswarm/services/stripe_elements.js.coffee

```diff
@@ -19,6 +19,11 @@ Darkswarm.factory 'StripeElements', ($rootScope, Messages) ->
           secrets.cc_type = @mapTokenApiCardBrand(response.token.card.brand)
           secrets.card = response.token.card
           submit()
+      .catch (response) =>
+        # Stripe handles errors in the response above. This code may never be
+        # reached. But if we get here, we want to know.
+        @reportError(response, t("js.stripe_elements.unknown_error_from_stripe"))
+        throw response
```

Comment:
In my tests, a `throw` didn't trigger any Bugsnag events. And Bugsnag's documentation mentions notifying explicitly:
https://docs.bugsnag.com/platforms/javascript/reporting-handled-errors/#handled-promise-rejections